### ISSUE TITLE
Inquiry

### ISSUE THREAD

**Issue by jeffreychaouc** [DhirenShivdasani/NBA-player-prop-analysis#1]
Hi! this repo caught my eye and I see that you're auto updating players' stats. I'm an avid nba fan, is it ok if we could talk sometime about how to run this?

**Comment by DhirenShivdasani** [comment]
yes ofc, sorry i am still figuring out some issues with this but i can def find some time to talk or upload instructions on how to run this. For now here is the link to the app: https://nba-player-prop-analysis-xaokjc9orfssctbbferycf.streamlit.app/

**Comment by jeffreychaouc** [comment]
ok cool my discord is : jchaodubs 
Thanks for sending the link over, i'm pretty crammed with school atm, but i'd like to contribute down the line
